obsah/__init__.py

```
@@ -36,6 +36,48 @@
 
+class AppendUniqueAction(argparse.Action):
+    """
+    Custom argparse action that appends values but ensures uniqueness.
+    Similar to action='append', but duplicate values are ignored.
+    """
+    def __init__(self, option_strings, dest, default=None, **kwargs):
+        if default is None:
+            default = []
+        super().__init__(option_strings, dest, default=default, **kwargs)
+
+    def __call__(self, parser, namespace, values, option_string=None):
```

Ruff (0.14.4)

Unused method argument: `parser`

(ARG002)